include/evmc/evmc.h

```diff
@@ -659,7 +659,9 @@ enum evmc_revision
     EVMC_TANGERINE_WHISTLE = 2,
     EVMC_SPURIOUS_DRAGON = 3,
     EVMC_BYZANTIUM = 4,
-    EVMC_CONSTANTINOPLE = 5
+    EVMC_CONSTANTINOPLE = 5,
+
+    EVMC_LATEST_REVISON = EVMC_CONSTANTINOPLE
```

Comment:
I wonder if this should be rather `LATEST_STABLE_REVISION` and point to `BYZANTIUM`?

Reply:
Not sure, probably yes. Would you use it anywhere in Hera?

Maybe name it `DEFAULT_REVISION`?